### ISSUE TITLE
Can't compile Yara 4.2.3 using MINGW when with-debug-verbosity is used.

### ISSUE THREAD

**Issue by CatyGreen** [VirusTotal/yara#1817]
**Describe the bug**
Can't compile Yara 4.2.3 using MINGW when with-debug-verbosity is used.
The following error appears various times in different lines of hash.c:

```
modules/hash/hash.c: In function 'get_from_cache':
modules/hash/hash.c:121:23:error:expected ')' before 'PRIi64'
121      " - %s (offset = % "PRI64 " length = % " PRIi64 ") {} = %p\n",
```

If I remove with-debug-verbosity it compiles fine.

**To Reproduce**
./configure --with-debug-verbosity=1 && make

**Expected behavior**
It should compile well.

**Please complete the following information:**
 - OS: MINGW64 (MSYS2)
 - YARA version: 4.2.3

**Comment by wxsBSD** [comment]
I think this was fixed in master (https://github.com/VirusTotal/yara/pull/1769/commits) but never merged to 4.2.x branch. Can you build off master?

**Comment by CatyGreen** [comment]
Thanks! With that fix I've been able to compile it.
